core/src/main/java/net/opentsdb/aura/metrics/meta/NewDocStore.java

```diff
@@ -137,9 +173,11 @@ public void add(final String[] tagKeys, final String[] tagValues, long key) {
     final int index;
     if (resets.isEmpty()) {
       index = currentIndex++;
+      m_maxDocId.set(index, tags);
     } else {
       index = resets.first();
       resets.remove(index);
+      m_freeDocs.set(resets.getLongCardinality(), tags);
```

Comment:
In theory, these metrics can have sparse data if no doc is added.

Reply:
Yup :)